showcase/app/templates/components/app-side-nav.hbs

```diff
@@ -13,68 +13,35 @@
   <Shw::Text::H3>With generic content</Shw::Text::H3>
 
   <Shw::Flex as |SF|>
-    <SF.Item @label="Header + Body + Footer, base invocation">
+    <SF.Item @label="Body base invocation">
       <div class="shw-component-sim-app-side-nav-root-container">
         <Hds::AppSideNav @isResponsive={{false}}>
-          <:header>
-            <Shw::Placeholder @text="header" />
-          </:header>
-          <:body>
-            <Shw::Placeholder @text="body" />
-          </:body>
-          <:footer>
-            <Shw::Placeholder @text="footer" />
-          </:footer>
+          <Shw::Placeholder @text="body" />
         </Hds::AppSideNav>
       </div>
     </SF.Item>
 
-    <SF.Item @label="Header + Body + Footer, heights declared">
+    <SF.Item @label="Body height declared">
       <div class="shw-component-sim-app-side-nav-root-container">
         <Hds::AppSideNav @isResponsive={{false}}>
-          <:header>
-            <Shw::Placeholder @text="header" @height="72px" />
-          </:header>
-          <:body>
-            <Shw::Placeholder @text="body" @height="300px" />
-          </:body>
-          <:footer>
-            <Shw::Placeholder @text="footer" @height="36px" />
-          </:footer>
+          <Shw::Placeholder @text="body" @height="300px" />
         </Hds::AppSideNav>
       </div>
     </SF.Item>
 
-    <SF.Item @label="Header (with extra) + Body (with extra) + Footer">
+    <SF.Item @label="Body (with extra)">
```

Comment:
[suggestion] at this point, this is not a relevant use case (I would leave only two cases: the one with the generic content and the one with the scrollable body, to test the overflow)